app-conf/elephant.conf

```diff
@@ -20,4 +20,4 @@ db_password=""
 
 # Additional Configuration
 # Check https://www.playframework.com/documentation/2.2.x/ProductionConfiguration
-jvm_args="-Devolutionplugin=enabled -DapplyEvolutions.default=true"
+jvm_args="-Devolutionplugin=enabled -DapplyEvolutions.default=true -jvm-debug 9999"
```

Comment:
I think it is good to keep it. It will help developers easily open a port to debug Dr. Elephant. There shouldn't be a problem retaining it.